package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "minimalcss",
-  "version": "0.8.2",
+  "version": "0.9.0",
```

Comment:
Please don't change the version. It's my job :)
Also, it's automated with npm scripts when I make the release.